src/gatsby-node.js

```diff
@@ -47,25 +47,37 @@ exports.sourceNodes = (
     return nodeData
   }
 
-  const { groupUrlName, ...apiOptions } = configOptions
+  const { groupUrlName, eventsOptions: paramEventsOptions, ...apiOptions } = configOptions
+  const eventsOptions = paramEventsOptions?paramEventsOptions:[apiOptions];
+
   // Convert the options object into a query string
   const queryStringOptions = queryString.stringify(apiOptions)
 
-  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`
-  const apiEventsUrl = `https://api.meetup.com/${groupUrlName}/events?${queryStringOptions}`
+  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`;
+
+  const allApiEventsUrl = eventsOptions.map(eventApiOptions => {
+    // Convert the options object into a query string
+    const queryStringEventOptions = queryString.stringify(eventApiOptions)
+    return `https://api.meetup.com/${groupUrlName}/events?${queryStringEventOptions}`;
+  });
+
+  const allApiUrls = [
+    apiGroupUrl,
+    ...allApiEventsUrl
+  ];
 
   // Gatsby expects sourceNodes to return a promise
   return (
     // Fetch a response from the apiUrl
-    Promise.all([fetch(apiGroupUrl), fetch(apiEventsUrl)])
+    Promise.all(allApiUrls.map(fetch))
```

Comment:
Tested your PR locally, and this line threw a pretty cryptic error:

`TypeError: Cannot use 'in' operator to search for 'signal' in 0`

Managed to fix it with the following change:

```diff
- Promise.all(allApiUrls.map(fetch))
+ Promise.all(allApiUrls.map(url => fetch(url))
```

I reproduced this on REPL.it: https://repl.it/repls/DefinitiveValuableRepositories

No idea why it throws an error though.

Reply:
Got my answer! https://twitter.com/kraynl/status/1213057654609907712

> Shorthand version calls fetch with the index of the item as a second argument. Fetch requires an object for its second arg, it probably does not like the integer. Same problem with parseInt and the base changing for every element!

Reply:
```suggestion
    Promise.all(allApiUrls.map(url => fetch(url))
```

Reply:
suggestion applied. Thanks by verify this error.